server.js

```diff
@@ -116,7 +124,7 @@ app.post("/callAnswered", async (req, res) => {
 app.get("/endPSTNCall", async (req, res) => {
   console.log("Hanging up PSTN call");
   try {
-    session_id = await getSessionId();
+    await getSessionId();
```

Comment:
So what is the point of the `getSessionId()`? Does it create the session

Reply:
Yes